stdlib/public/core/Character.swift

```diff
@@ -120,7 +120,7 @@ public struct Character :
   @warn_unused_result
   static func _smallSize(value: UInt64) -> Int {
     var mask: UInt64 = 0xFF
-    for var i = 0; i < 8; i += 1 {
+    for i in 0..<8 {
```

Comment:
There’s no official guide.  I will say that the standard library should remain internally consistent, I don’t see any compelling reason to change what we’re doing with range operator spacing.